src/skycoin/config.go

```diff
@@ -134,7 +137,7 @@ type NodeConfig struct {
 	DBReadOnly  bool   `mapstructure:"db_read_only"`
 	Arbitrating bool   `mapstructure:"arbitrating"`
 	LogToFile   bool   `mapstructure:"log_to_file"`
-	Version     bool   `mapstructure:"version"` // show node version
+	ShowVersion bool   `mapstructure:"show_version"` // show node version
```

Comment:
This shouldn't be changed. -version is a standard way to print an application's version.  The version wouldn't be configurable from the command line

Reply:
And this would always be false by default